src/battle/area_flo/5B1A10.c

```diff
@@ -14,4 +14,48 @@
 
 #include "common/StartRumbleWithParams.inc.c"
 
-INCLUDE_ASM(s32, "battle/area_flo/5B1A10", func_80218808_5B1CB8);
+ApiStatus func_80218808_5B1CB8(Evt* script, s32 isInitialCall) {
+    Actor* actor;
+    s32 actorType;
+    s32 value;
+
+    Bytecode* readPos = script->ptrReadPos;
+    ActorID actorId = evt_get_variable(script, *readPos++);
+
+    if (actorId == -0x7F) {
```

Comment:
this is checking if the actor id is self. see BindNextTurn for an example (and please call these `actorID`)